dask/dataframe/methods.py

```diff
@@ -162,15 +165,18 @@ def describe_numeric_aggregate(stats, name=None, is_timedelta_col=False):
         max = pd.to_timedelta(max)
         q = q.apply(lambda x: pd.to_timedelta(x))
 
-    part1 = typ([count, mean, std, min], index=["count", "mean", "std", "min"])
+    part1 = typ(
+        [count.astype("float64"), mean, std, min], index=["count", "mean", "std", "min"]
+    )
+
     q.index = ["{0:g}%".format(l * 100) for l in q.index.tolist()]
-    if isinstance(q, pd.Series) and typ == pd.DataFrame:
+    if is_series_like(q) and is_dataframe_like(typ):
         q = q.to_frame()
     part3 = typ([max], index=["max"])
 
-    result = pd.concat([part1, q, part3], **concat_kwargs)
+    result = DD.concat([part1, q, part3], **concat_kwargs)
```

Comment:
Perhaps use the concat function defined in this module instead

Reply:
Using the concat function in this module, but to preserve `concat_kwargs` added it to the `concat` method too. Adding support for cudf backend too as a stale function param.